site/content/en/docs/Guides/metrics.md

```diff
@@ -19,7 +19,6 @@ If you are running a [Prometheus](https://prometheus.io/) instance you just need
 
 {{% feature publishVersion="1.26.0" %}}
 If your Prometheus metrics collection agent requires that you scrape from the pods directly(such as with [Google Cloud Managed Prometheus](https://cloud.google.com/stackdriver/docs/managed-prometheus)), then the metrics ports for the controller and allocator will both be named `http` and exposed on `8080`. In the case of the allocator, the port name and number can be overriden with the `agones.allocator.serviceMetrics.http.portName` and `agones.allocator.serviceMetrics.http.port` helm chart values.
-{{% /feature %}}
```

Comment:
Looks like the test is failing because you've removed the bottom tag for `feature`, but not the top.

Reply:
Ya got it, i started searching with this </% /feature %>  so I missed for this one.

Reply:
Looks like you fixed the below one, but not this one.